learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -143,21 +162,29 @@ public distinct service class CovidData {
 }
 ```
 
-Here, you are creating the `CovidData` service type to represent the GraphQL `Object` type that represents an entry in
-the data set. Each resource method in this service represents a field of the GraphQL object type.  The return type of
+In this code:
+- The endpoint returns the number of cases in thousands. Therefore, a service type is used to define the output object type, and inside the service type, each resource function will return the original value
+divided by 1000.
+- You are creating the `CovidData` service type to represent the GraphQL `Object` type, which represents an entry in
+the data set. 
+- Each resource method in this service represents a field of the GraphQL object type.  The return type of
 the resource method is the type of the field.
-The resource methods returning `isoCode` and the `country` have the return type `string`, which means these fields
+- The resource methods returning `isoCode` and the `country` have the return type `string`, which means these fields
 cannot be `null` in the GraphQL response. In other words, these fields have `NON_NULL` types. (In GraphQL, these are
-represented by the exclamation mark `!`. E.g., `String!`). However, the resource methods returning numbers can return `null`
-values. Therefore, the type of fields represented by those resource methods is nullable.
+represented by the exclamation mark `!`. E.g., `String!`). However, the resource methods returning numbers can return `null` values. Therefore, the type of fields represented by those resource methods is nullable.
+- The `decimal` type is used as a return type. This will add a `Scalar` type named `Decimal` to the GraphQL schema, which will be generated by the Ballerina GraphQL package.
 
-> **Note:**  The `decimal` type is used as a return type. This will add a `Scalar` type named `Decimal` to the GraphQL
-> schema, which will be generated by the Ballerina GraphQL package.
+## Create the service
 
-### Write the GraphQL service
+Now, you are all set to write the GraphQL service.
 
-Now you are all set to write the GraphQL service. To write the service, you need to create a GraphQL listener object.
-To create a listener object, you need to import the Ballerina GraphQL package.
+### Create the listener object
+
+To write the service, you need to create a GraphQL listener object.
+
+#### Import the Ballerina GraphQL package
+
+To import the Ballerina GraphQL package for creating the listener object, add the code below to the API template file (i.e., `main.bal`). 
```

Comment:
I think the titles are incorrect. The code snippet is not for importing the package.